pyartcd/pyartcd/pipelines/comment_to_pr.py

```diff
@@ -281,16 +280,22 @@ async def _builds_from_job(self):
         url = f"{ART_DASH_API_ENDPOINT}/builds?jenkins_build_url__icontains={self.job_name}/{self.job_id}"
         self._logger.info(f"Querying ART Dash server with url: {url}")
 
-        response = requests.get(url)
-        if response.status_code != 200:
-            self._logger.error(f"ART DASH Server error. Status code: {response.status_code}")
-            sys.exit(1)
+        # Fetch data
+        async with aiohttp.ClientSession() as session:
+            self._logger.info("Fetching url %s", url)
+            async with session.get(url) as url_response:
+                try:
+                    url_response.raise_for_status()
+                    # response.json() will throw an exception as the content type is text/plain
```

Comment:
Just curious, setting the correct header `Content-Type: application/json` would to the trick? Otherwise, I think something needs to be done on the endpoint side